.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Not better once a week, like in OS Hz?

Reply:
In HZ/OS we have it once a week to avoid new PRs for libraries that release daily (aws, mockito..). I don't think we have this issue here.